BE/src/Providers/ModEval/GQL_Queries/Fields/Fields.ts

```diff
@@ -187,38 +187,15 @@ export const CreateTestMasterQuery = () => `
     }
 `;
 
-export function CreateReviewedPRField(owner: string, repoName: string, after: string | null = null): string {
-    const afterClause = after ? `, after: "${after}"` : "";
+export function CreatePRMergesField(owner: string, repoName: string): string {
     return `
     {
         repository(owner: "${owner}", name: "${repoName}") {
-            pullRequests(first: 100, states: MERGED${afterClause}) {
-                pageInfo {
-                    endCursor
-                    hasNextPage
-                }
+            pullRequests(first: 100, states: MERGED) {
```

Comment:
This is still taking around 8 seconds. What are you thoughts on reducing this to 20?